app/react-components/Mutation.js

```diff
@@ -186,6 +186,8 @@ let Mutation = (() => {
         .map(k => ({project_id: k, ...cancerDistData[k]}))
         .sort((a, b) => b.freq - a.freq);
 
+      const distinctCases = sortedCancerDistData.reduce((acc, d) => [...acc, ...d.cases], []);
```

Comment:
didn't catch your whole verbal conversation, but re-looked at the query and yeah I think this is correct. Pulling case_ids from the occurrence, then making them extra-distinct if they weren't already by using Set(). It's not like the Most Frequently Mutated Genes case counts where some of the cases could have no ssm.